script.py

```diff
@@ -0,0 +1,29 @@
+import time
```

Comment:
What is this?

Reply:
it's a file that shouldn't have been included in the commit ! I removed it